src/Microsoft.CodeAnalysis.NetAnalyzers/src/Microsoft.CodeAnalysis.NetAnalyzers/Microsoft.CodeQuality.Analyzers/Maintainability/UseCrossPlatformIntrinsicsAnalyzer.cs

```diff
@@ -140,169 +204,357 @@ private void OnCompilationStart(CompilationStartAnalysisContext context)
+                AddUnaryMethods(methodSymbols, "ConvertToInt32RoundToZero", armAdvSimdTypeSymbolForMethods, RuleKind.ConvertToInt32);
+                AddUnaryMethods(methodSymbols, "ConvertToInt32RoundToZeroScalar", armAdvSimdTypeSymbolForMethods, RuleKind.ConvertToInt32);
+                // Extract maps to GetElement - needs index parameter handling in fixer
+                // TODO: Implement Extract → GetElement transformation
+                AddUnaryMethods(methodSymbols, "Floor", armAdvSimdTypeSymbolForMethods, RuleKind.Floor);
+                AddUnaryMethods(methodSymbols, "FloorScalar", armAdvSimdTypeSymbolForMethods, RuleKind.Floor);
+                AddTernaryMethods(methodSymbols, "FusedMultiplyAdd", armAdvSimdTypeSymbolForMethods, RuleKind.FusedMultiplyAdd);
+                // FusedMultiplySubtract and variants need parameter negation/reordering in fixer
+                // TODO: Implement FusedMultiplySubtract transformations
+                // Insert maps to WithElement - needs index parameter handling in fixer
+                // TODO: Implement Insert → WithElement transformation
+                AddBinaryMethods(methodSymbols, "Max", armAdvSimdTypeSymbolForMethods, RuleKind.Max);
+                AddBinaryMethods(methodSymbols, "Min", armAdvSimdTypeSymbolForMethods, RuleKind.Min);
+                // Note: Negate is already registered as op_UnaryNegation above, so we don't register it here
+                // Note: MultiplyAdd(x, y, z) expands to (y * z) + x - needs operator expansion
+                // TODO: Implement MultiplyAdd → operator expansion transformation
+                // Note: OrNot expands to two operators - needs operator expansion  
+                // TODO: Implement OrNot → operator expansion transformation
+                AddUnaryMethods(methodSymbols, "RoundToNearest", armAdvSimdTypeSymbolForMethods, RuleKind.Round);
+                AddUnaryMethods(methodSymbols, "RoundToNegativeInfinity", armAdvSimdTypeSymbolForMethods, RuleKind.Floor);
+                AddUnaryMethods(methodSymbols, "RoundToPositiveInfinity", armAdvSimdTypeSymbolForMethods, RuleKind.Ceiling);
+                AddUnaryMethods(methodSymbols, "RoundToZero", armAdvSimdTypeSymbolForMethods, RuleKind.Truncate);
+                // Note: Sqrt is already registered above as operator method
+                AddLoadMethods(methodSymbols, "LoadVector64", armAdvSimdTypeSymbolForMethods, RuleKind.Load);
+                AddLoadMethods(methodSymbols, "LoadVector128", armAdvSimdTypeSymbolForMethods, RuleKind.Load);
+                AddStoreMethods(methodSymbols, "Store", armAdvSimdTypeSymbolForMethods, RuleKind.Store);
+                // VectorTableLookup is 1-to-1 with Shuffle for byte/sbyte
+                AddBinaryMethods(methodSymbols, "VectorTableLookup", armAdvSimdTypeSymbolForMethods, RuleKind.Shuffle, [SpecialType.System_Byte, SpecialType.System_SByte]);
```

Comment:
The filter here is unnecessary as `VectorTableLookup` only exists for byte/sbyte